search-commons/src/main/java/no/unit/nva/search/resource/SimplifiedResourceModelMutator.java

```diff
@@ -0,0 +1,300 @@
+                                        .path(PUBLICATION_DATE)
+                                        .path(YEAR)
+                                        .textValue(),
+                                source.path(ENTITY_DESCRIPTION)
+                                        .path(PUBLICATION_DATE)
+                                        .path(MONTH)
+                                        .textValue(),
+                                source.path(ENTITY_DESCRIPTION)
+                                        .path(PUBLICATION_DATE)
+                                        .path(DAY)
+                                        .textValue()))
+                .withContributorsPreview(mutateContributorsPreview(source))
+                .withContributorsCount(
+                        source.path(ENTITY_DESCRIPTION).path(CONTRIBUTORS_COUNT).asInt())
+                .withPublishingDetails(mutatePublishingDetails(source))
+                .withOtherIdentifiers(mutateOtherIdentifiers(source))
+                .withRecordMetadata(mutateRecordMetadata(source))
+                .build();
+    }
+
+    private List<String> jsonNodeMapToValueList(JsonNode source) throws IOException {
+        Map<String, String> map = objectMapper.convertValue(source, Map.class);
+        return map.values().stream().collect(Collectors.toList());
+    }
+
+    private OtherIdentifiers mutateOtherIdentifiers(JsonNode source) throws IOException {
+        var issns =
+                Stream.of(
+                                source.path(ENTITY_DESCRIPTION)
+                                        .path(REFERENCE)
+                                        .path(PUBLICATION_CONTEXT)
+                                        .path(ONLINE_ISSN)
+                                        .textValue(),
+                                source.path(ENTITY_DESCRIPTION)
+                                        .path(REFERENCE)
+                                        .path(PUBLICATION_CONTEXT)
+                                        .path(PRINT_ISSN)
+                                        .textValue(),
+                                source.path(ENTITY_DESCRIPTION)
+                                        .path(REFERENCE)
+                                        .path(PUBLICATION_CONTEXT)
+                                        .path(SERIES)
+                                        .path(ONLINE_ISSN)
+                                        .textValue(),
+                                source.path(ENTITY_DESCRIPTION)
+                                        .path(REFERENCE)
+                                        .path(PUBLICATION_CONTEXT)
+                                        .path(SERIES)
+                                        .path(PRINT_ISSN)
+                                        .textValue())
+                        .filter(Objects::nonNull)
+                        .collect(Collectors.toSet());
+
+        var isbnsInSource =
```

Comment:
Unfortunately, we also have ISBN in Manifestations of Artistic

Reply:
@brinxmat Can you help me with the jsonpath for those? cant find them